common/app/model/meta.scala

```diff
@@ -196,7 +197,7 @@ object MetaData {
       webUrl = apiContent.webUrl,
       maybeSectionId,
       Pillar(apiContent),
-      format = Some(contentFormat),
+      contentFormat,
```

Comment:
The format information should always be available, hence the decision to not make this an `Option`. However, it might be the case that because it's not always needed we should return to using an `Option`, allowing us to pass `None` to places where we don't expect to need it e.g. in `Section.scala` and `Tag.scala`